docs/service-catalog/03-01-resources.md

```diff
@@ -25,6 +26,9 @@ This document includes an overview of resources that the Kyma Service Catalog pr
 
 * **Secret** is a basic resource to transfer credentials or configuration details that the application uses to consume a ServiceInstance. The service binding process leads to the creation of a Secret.
 
-* **ServiceBindingUsage** is a Kyma custom resource that allows the ServiceBindingUsage controller to inject Secrets into a given application.
 
-* **UsageKind** is a Kyma custom resource that defines which resources can be bound with the ServiceBinding and how to bind them.
+These are the Service Catalog resources that Kyma provides:
+
+* [**ServiceBindingUsage**](#custom-resource-service-binding-usage) is a Kyma custom resource that allows the ServiceBindingUsage controller to inject Secrets into a given application.
```

Comment:
```suggestion
* [**ServiceBindingUsage**](#custom-resource-service-binding-usage) - a Kyma custom resource that allows the ServiceBindingUsage controller to inject Secrets into a given application.
```
I'd change the `is`'s to `-`'s, as you already introduce the resources in line 30 and then list them :)